### PR TITLE
[ADD] fieldservice_equipment_type

### PR DESCRIPTION
**Pull request by BojanOD**
This module serves as an enhancement that adds a type field to the Field Service Equipment model.

### CODE REVIEW

**Review by SilvioC2C — APPROVED** [review]

**Comment by max3903** [comment]
@BojanOD Can you add the `res.config.setting` option to install this module in the `fieldservice` module?

**Review by max3903 — APPROVED** [review]

File: fieldservice_equipment_type/models/fsm_equipment_type.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+from odoo import fields, models
+
+
+class FSMEquipmentType(models.Model):
+    _name = "fsm.equipment.type"
+    _description = "FSM Equipment Type"
+
+    name = fields.Char(required=True)
+    code = fields.Char(required=True)
```
Comment (hparfr) [inline fieldservice_equipment_type/models/fsm_equipment_type.py:12]: Why this field `code` has to be required ?

Comment (SMaciasOSI) [reply fieldservice_equipment_type/models/fsm_equipment_type.py:12]: @BojanOD Could you help with this question?

**Pull request merged**